app/views/admin/tags/edit.html.erb

```diff
@@ -5,7 +5,7 @@
   </div>
 
   <div class="ml-auto">
-    <a href="<%= tag_path(@tag.name) %>" class="crayons-btn crayons-btn--outlined" target="_blank" rel="noopener noreferer">View</a>
```

Comment:
I also considered just using the path helper. After all the conversation, I mostly wanted to use the same code paths to generate all the links. I would love it if we used the routing helpers built into Rails, but since I don't know the reason the custom `URL` generators exist I didn't know if it was safe to swap to the Rails helpers.

Reply:
The other thing was that I don't know why we use `<a href=...>` instead of the Rails `link_to` helper, but for now I'm assuming there was a reason for it.

Reply:
I'm definitely in favor of using `link_to` and the builting tag path helper :) I think some are more used to typing `a href` than using `link_to`